source/modulo_controllers/src/RobotControllerInterface.cpp

```diff
@@ -60,6 +63,12 @@ rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn RobotC
     auto urdf_path = "/tmp/" + hardware_name_ + "_" + timestamp.str();
     try {
       robot_model::Model::create_urdf_from_string(get_parameter_value<std::string>("robot_description"), urdf_path);
+      if (load_geometries_) {
+        robot_ = std::make_shared<robot_model::Model>(hardware_name_, urdf_path, [](const std::string& package_name) {
+          return ament_index_cpp::get_package_share_directory(package_name) + "/";
+        });
+        RCLCPP_DEBUG(get_node()->get_logger(), "Generated robot model with collision features");
+      }
```

Comment:
I'm a bit confused, shouldn't the `robot_ = ...` that comes after this line be in the `else`?